src/components/myOrders/MyOrderItem.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {styles} from './styles';
+import {Image, Text, View} from 'react-native';
+import DateTimeUtils from '../../utils/DateTimeUtils';
+
+type MyOrderItemProp = {
+  name: string;
+  title: string;
+  date: string;
+  imagePath: string;
+  total: string;
+  currency: string;
+  orderIncluded: [];
+};
+
+const MyOrderItem: React.FC<MyOrderItemProp> = ({
+  name,
+  title,
+  date,
+  imagePath,
+  total,
+  currency,
+  orderIncluded,
+}) => (
+  <View style={styles.priceItemsElements}>
+    {orderIncluded && (
```

Comment:
It seems, that in an a falsy case there'll be rendered nothing. Is it an expected behaviour?